packages/api/src/controllers/customAnimalTypeController.js

```diff
@@ -30,6 +33,43 @@ const customAnimalTypeController = {
       }
     };
   },
+
+  addCustomAnimalType() {
```

Comment:
I saw in Anto's add animals PR that she was immediately allowing the possibility of adding an array of animals.

Should this allow an array of types? I am just thinking about how the designs show you can add multiple types at once.
![Screenshot 2024-01-22 at 11 33 48 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/494a9593-4881-4c1c-8833-f1c01e11c70b)

Reply:
I don't think the UI will allow adding multiple types at once. You can definitely add multiple animals at once as reflected in this screenshot, but for types I think the way it's going to work is the "Animal type" dropdown is actually going to be an autocomplete, and you're going to be able to type in something that doesn't exist and click on "Add new" or something similar. So you'd add them one by one